Doc/library/runpy.rst

```diff
@@ -165,8 +166,8 @@ The :mod:`runpy` module provides two functions:
    .. versionchanged:: 3.4
       Updated to take advantage of the module spec feature added by
       :pep:`451`. This allows ``__cached__`` to be set correctly in the
-      case where ``__main__`` is imported from a valid sys.path entry rather
-      than being executed directly.
+      case where ``__main__`` is imported from a valid :data:`sys.path` entry
+      rather than being executed directly.
```

Comment:
Same as above: it doesn't matter if the line length goes _slightly_ over 80 characters; better to aim for a smaller diff:

```suggestion
      case where ``__main__`` is imported from a valid :data:`sys.path` entry rather
      than being executed directly.
```